components/core/header/nav-bar/nav-bar-items.js

```diff
@@ -16,6 +16,10 @@ export default Object.freeze({
             i18nKey: 'codeOfConduct',
             value: '/about/code-of-conduct',
         },
+        {
```

Comment:
staff link will only be in the footer of each page. need to remove from navbar

Reply:
I assume there are only 2 commits that should be left at this stage but there are 8 commits now. Try using the git command `git rebase -i <commit hash>` to eliminate the excessive commits. 

reference: https://dev.to/lydiahallie/cs-visualized-useful-git-commands-37p1#rebase (read the `interactive rebase` section)

Reply:
Don't worry. It's totally correct to do so!

Since the remote branch has difference with your local one, regular push leads you to warning message and block the action. Once you confirmed your local branch is correct, then it's safe to force push it to remote.

Reply:
That's interesting.
And, rebase function really makes the commit history more clear and precise.
Thanks for your explanation.😃